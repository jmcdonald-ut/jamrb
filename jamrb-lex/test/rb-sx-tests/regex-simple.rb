/hay/ =~ 'haystack'
/y/ =~ 'haystack'

/\s\u{6771 4eac 90fd}/.match("Go to 東京都")
/[0-9a-f]/.match('9f')
