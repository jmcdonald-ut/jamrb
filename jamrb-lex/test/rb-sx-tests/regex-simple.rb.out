((Token ((1 0) on_regexp_beg "/"))
 (Token ((1 1) on_tstring_content "hay"))
 (Token ((1 4) on_regexp_end "/"))
 (Token ((1 5) on_sp " "))
 (Token ((1 6) on_op "=~"))
 (Token ((1 8) on_sp " "))
 (Token ((1 9) on_tstring_beg "'"))
 (Token ((1 10) on_tstring_content "haystack"))
 (Token ((1 18) on_tstring_end "'"))
 (Token ((1 19) on_nl "\n"))
 (Token ((2 0) on_regexp_beg "/"))
 (Token ((2 1) on_tstring_content "y"))
 (Token ((2 2) on_regexp_end "/"))
 (Token ((2 3) on_sp " "))
 (Token ((2 4) on_op "=~"))
 (Token ((2 6) on_sp " "))
 (Token ((2 7) on_tstring_beg "'"))
 (Token ((2 8) on_tstring_content "haystack"))
 (Token ((2 16) on_tstring_end "'"))
 (Token ((2 17) on_nl "\n"))
 (Token ((3 0) on_ignored_nl "\n"))
 (Token ((4 0) on_regexp_beg "/"))
 (Token ((4 1) on_tstring_content "\\\\s\\\\u{6771 4eac 90fd}"))
 (Token ((4 21) on_regexp_end "/"))
 (Token ((4 22) on_period "."))
 (Token ((4 23) on_ident "match"))
 (Token ((4 28) on_lparen "("))
 (Token ((4 29) on_tstring_beg "\""))
 (Token ((4 30) on_tstring_content "Go to 東京都"))
 (Token ((4 45) on_tstring_end "\""))
 (Token ((4 46) on_rparen ")"))
 (Token ((4 47) on_nl "\n"))
 (Token ((5 0) on_regexp_beg "/"))
 (Token ((5 1) on_tstring_content "[0-9a-f]"))
 (Token ((5 9) on_regexp_end "/"))
 (Token ((5 10) on_period "."))
 (Token ((5 11) on_ident "match"))
 (Token ((5 16) on_lparen "("))
 (Token ((5 17) on_tstring_beg "'"))
 (Token ((5 18) on_tstring_content "9f"))
 (Token ((5 20) on_tstring_end "'"))
 (Token ((5 21) on_rparen ")"))
 (Token ((5 22) on_nl "\n")))
